src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedNamedTypeSymbol.cs

```diff
@@ -174,7 +174,9 @@ internal override bool IsMetadataSealed
                 return _underlyingType.IsMetadataSealed;
             }
         }
-        
+
+        public override bool AreLocalsZeroed => _underlyingType.AreLocalsZeroed;
```

Comment:
>public override bool AreLocalsZeroed => _underlyingType.AreLocalsZeroed; [](start = 8, length = 72)

I think it would be better to implement this in leaf classes, then we can see why implementation is needed and why is it correct. Also we can make sure that the test coverage is adequate.